src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -109,20 +186,21 @@ private void assertRecordsInTable() {
 
   private static Stream<Arguments> prepareData() {
     return Stream.of(
-        Arguments.of(
-            "Primitive JSON with schema",
-            primitiveJsonWithSchemaExample,
-            new DescribeTableRow[] {
-              new DescribeTableRow("ID_INT8", "NUMBER(10,0)"),
-              new DescribeTableRow("ID_INT16", "NUMBER(10,0)"),
-              new DescribeTableRow("ID_INT32", "NUMBER(10,0)"),
-              new DescribeTableRow("ID_INT64", "NUMBER(19,0)"),
-              new DescribeTableRow("DESCRIPTION", "VARCHAR(16777216)"),
-              new DescribeTableRow("RATING_FLOAT32", "FLOAT"),
-              new DescribeTableRow("RATING_FLOAT64", "FLOAT"),
-              new DescribeTableRow("APPROVAL", "BOOLEAN")
-            },
-            true),
+        // READING SCHEMA FROM A RECORD IS NOT YET SUPPORTED.
```

Comment:
unfinished test?

Reply:
I temporary broke schema evolution based on schema. Now it works again.